pr_check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Run unit tests first since it activates/deactives its own virtual env
+source unit_test.sh
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="quickstarts"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="quickstarts"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/quickstarts"
+
+IQE_PLUGINS="e2e"
+IQE_MARKER_EXPRESSION="smoke"
+IQE_FILTER_EXPRESSION=""
+IQE_CJI_TIMEOUT="30m"
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+source $CICD_ROOT/build.sh
+source $CICD_ROOT/deploy_ephemeral_env.sh
+
+# Deploy HBI app required for the smoke tests
```

Comment:
Is HBI a dependency of quickstarts? Also, you aren't smoke testing at all yet.

Reply:
Definitely not. I forgot to remove it while looking at the examples in the docs.